service.ifttt/service.py

```diff
@@ -0,0 +1,112 @@
+import os, sys, re
+import xbmc, xbmcaddon, xbmcgui
+import json, urllib2
+
+__addon__ = xbmcaddon.Addon()
+__addonversion__ = __addon__.getAddonInfo('version')
+__addonid__ = __addon__.getAddonInfo('id')
+__addonname__ = __addon__.getAddonInfo('name')
+__addonPath__ = __addon__.getAddonInfo('path')
+__addonResourcePath__ = xbmc.translatePath(os.path.join(__addonPath__, 'resources', 'lib'))
+__addonIconFile__ = xbmc.translatePath(os.path.join(__addonPath__, 'icon.png'))
+sys.path.append(__addonResourcePath__)
```

Comment:
Please use relative imports if possible instead of the append to the sys.path

Reply:
Im not really sure how to change this can you provide an example.

Reply:
Just do `from resources.lib import whateveryouwant`

You might need a placeholder/empty  `__init__.py` file in resources, and another one in lib

Reply:
Still no idea what youre asking, this is a standard bit of code from the base templates I didnt write it and dont understand what youre saying, please be specific or point me to a sample project that does this as your expecting.

Reply:
See https://github.com/xbmc/repo-plugins/pull/3074#pullrequestreview-472262294 for a similar example